web/react/utils/channel_intro_mssages.jsx

```diff
@@ -214,3 +173,15 @@ export function createStandardIntroMessage(channel, showInviteModal) { //eslint-
         </div>
     );
 }
+
+function createSetHeaderButton(channel) { // eslint-disable-line react/no-multi-comp
```

Comment:
This is because you are! This is a valid react component: https://facebook.github.io/react/blog/#stateless-functional-components